src/components/Molecules/Task/index.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { useRef, useEffect, useState } from "react";
+import styled from "styled-components";
+import CheckBox from "../../Atoms/Checkbox/index";
+import Input from "../../Atoms/Input/index";
+import EditButton from "../../Atoms/EditButton/index";
+import COLOR from "../../../variables/color";
+import TEXT from "../../../variables/texts";
+
+const Task = (props) => {

```

Comment:
```suggestion
const Task = ({
  defaultValue,
  defaultEdit
  // その他にもいろいろ
}) => {
```
分割代入を使うと、どのようなpropsが存在するのかがわかりやすくなり見通しが良くなると思います。
参考： https://dezanari.com/react-component-props-object/